app/views/shared/_google_analytics_snippet.slim

```diff
@@ -0,0 +1,2 @@
+script
+  |  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){ (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o), m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m) })(window,document,'script','//www.google-analytics.com/analytics.js','ga'); ga('create', 'UA-26370633-3', 'auto'); ga('send', 'pageview'); 
```

Comment:
I might suggest we just merge this, given the change is also needed for Optimizely, and given that I can't actually figure out easily where the relevant ENV settings go (I see the relevant place in the three environment YMLs, I think, but that just refers to something outside the open source repo I think)?